configs/config-ccm-psk-dtls1_2.h

```diff
@@ -23,19 +23,20 @@
  * See README.txt for usage instructions.
  */
 
+#define MBEDTLS_PSA_CRYPTO_CONFIG_FILE "../configs/crypto-config-ccm-psk-tls1_2.h"
+
+#define MBEDTLS_PSA_CRYPTO_CONFIG
```

Comment:
Currently, `test-ref-configs.pl` runs the configurations `config-ccm-psk-tls1_2.h`, `config-ccm-psk-dtls1_2.h`, `config-suite-b.h` and `config-thread.h` twice: once as is, and once with `MBEDTLS_PSA_CRYPTO_C` + `MBEDTLS_USE_PSA_CRYPTO` enabled. The changes here only make sense for that second run. The non-PSA run doesn't make sense anymore and should be removed. Without this “two runs” logic, it no longer makes sense to have `test-ref-configs.pl` tweak the configuration, so PSA should be enabled here.

* Please add `#define MBEDTLS_PSA_CRYPTO_C` and `#define MBEDTLS_USE_PSA_CRYPTO` alongside `#define MBEDTLS_PSA_CRYPTO_CONFIG`.
* Please remove the two-configs logic from `test-ref-configs.pl`. (If you don't know Perl, someone can help.) (Note that there's also a two-configs logic when executing `ssl-opt.sh`, with/without traces, that part needs to stay.)